po/ja.po

```diff
@@ -18520,94 +18554,118 @@ msgid ""
+msgstr "`src/main.rs` ファイルは次のようになります。"
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"request error: {0}\""
-msgstr ""
+msgstr "\"request error: {0}\""
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"bad http response: {0}\""
-msgstr ""
+msgstr "\"bad http response: {0}\""
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"Checking {:#}\""
-msgstr ""
+msgstr "\"Checking {:#}\""
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"href\""
-msgstr ""
+msgstr "\"href\""
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"On {base_url:#}: ignored unparsable {href:?}: {err}\""
-msgstr ""
+msgstr "\"On {base_url:#}: ignored unparsable {href:?}: {err}\""
 
 #: src/exercises/concurrency/link-checker.md
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"https://www.google.org\""
-msgstr ""
+msgstr "\"https://www.google.org\""
 
 #: src/exercises/concurrency/link-checker.md
 msgid "\"Links: {links:#?}\""
-msgstr ""
+msgstr "\"Links: {links:#?}\""
 
 #: src/exercises/concurrency/link-checker.md
 msgid "\"Could not extract links: {err:#}\""
-msgstr ""
+msgstr "\"Could not extract links: {err:#}\""
 
 #: src/exercises/concurrency/link-checker.md
 msgid "Run the code in `src/main.rs` with"
-msgstr ""
+msgstr "`src/main.rs` 内のコードを、次のコマンドで実行します。"
 
 #: src/exercises/concurrency/link-checker.md
 msgid ""
 "Use threads to check the links in parallel: send the URLs to be checked to a "
 "channel and let a few threads check the URLs in parallel."
 msgstr ""
+"スレッドを使用してリンクを同時にチェックします。つまり、チェックする URL を"
+"チャンネルに送信し、いくつかのスレッドで同時に URL を確認します。"
 
 #: src/exercises/concurrency/link-checker.md
 msgid ""
 "Extend this to recursively extract links from all pages on the `www.google."
 "org` domain. Put an upper limit of 100 pages or so so that you don't end up "
 "being blocked by the site."
 msgstr ""
+"これを拡張して、`www.google.org` ドメインのすべてのページからリンクを再帰的に"
+"抽出します。サイトがブロックされないように、ページ数の上限を 100 程度に設定し"
+"ます。"
 
 #: src/exercises/concurrency/solutions-morning.md
 msgid "Concurrency Morning Exercise"
-msgstr ""
+msgstr "同時実行の午前のエクササイズ"
 
 #: src/exercises/concurrency/solutions-morning.md
 msgid "([back to exercise](dining-philosophers.md))"
-msgstr ""
+msgstr "（[演習に戻る](dining-philosophers.md)）"
 
 #: src/exercises/concurrency/solutions-morning.md
 msgid "\"{} is trying to eat\""
-msgstr ""
+msgstr "\"{} is trying to eat\""
 
 #: src/exercises/concurrency/solutions-morning.md
 msgid ""
 "// To avoid a deadlock, we have to break the symmetry\n"
 "        // somewhere. This will swap the forks without deinitializing\n"
 "        // either of them.\n"
 msgstr ""
+"// デッドロックを避けるために、どこかで対称性を\n"
+"        // 崩す必要があります。これにより、いずれのフォークも初期化解除するこ"
+"となく、フォークが\n"
+"        // スワップされます。\n"
```

Comment:
What should be replaced?

Reply:
"これにより、いずれのフォークも初期化解除することなく、フォークがスワップされます。" can be updated as "下記のコードでは、領域を開放することなく2つのフォークを交換します".

I believe this sentence describes the following code: "This will swap the forks without deinitializing either of them, so my suggestion is translating "this" as "下記のコード".  Also,  I am suggesting describing "deinitializing" with a longer sentence as "領域を開放することなく" rather than direct translation, i..e "初期化解除".